os/include/tinyara/testcase_drv.h

```diff
@@ -52,7 +52,26 @@
  *
  */
 
-#define TESTIOC_DRIVER_ANALOG            _TESTIOC(1)
+#define TESTIOC_ANALOG                          _TESTIOC(1)
+#define TESTIOC_CLOCK_ABSTIME2TICKS             _TESTIOC(2)
+#define TESTIOC_GET_SELF_TCB_PID                _TESTIOC(3)
+#define TESTIOC_GET_SIG_FINDACTION_ADD          _TESTIOC(4)
+#define TESTIOC_GET_TCB_ALIVE                   _TESTIOC(5)
+#define TESTIOC_GET_TCB_LOCKCOUNT               _TESTIOC(6)
+#define TESTIOC_GET_TCB_PID                     _TESTIOC(7)
+#define TESTIOC_GET_TCB_SIGPROCMASK             _TESTIOC(8)
+#define TESTIOC_GET_TCB_STACK_SIZE              _TESTIOC(9)
+#define TESTIOC_GET_TCB_TIMESLICE               _TESTIOC(10)
+#define TESTIOC_SCHED_FOREACH                   _TESTIOC(11)
+#define TESTIOC_TC_GROUP_ADD_FIND_REMOVE_CHILD  _TESTIOC(12)
```

Comment:
This define is too long. Please make it short.

Reply:
It's deleted. I will add next commit.